docs/rules/symbol-description.md

```diff
@@ -0,0 +1,61 @@
+# Require Symbol Description (symbol-description)
+
+The `Symbol` function may have optional description:
+
+```js
+var foo = Symbol("foo");
+
+var someString = 'bar';
+var bar = Symbol(someString);
+```
+
+
+Using `description` promotes easier debugging: when a symbol is logged the description is used:
+
+```js
+var foo = Symbol("foo");
+
+> console.log(foo);
+// Symbol(foo)
+```
+
+It may facilitate identifying symbols when one is observed during debugging.
+
+
+## Rule Details
+
+This rule allows enforcing or disallowing symbol descriptions.
```

Comment:
I'm not sure I like "allows" followed by "disallowing", seems a bit confusing. I think it's better to follow the language we used in other rules like `arrow-body-style`, `func-name`, etc. "This rule can enforce or disallow the use of braces around arrow function body."